test/unit/test_factory.cpp

```diff
@@ -2103,6 +2103,58 @@ TEST(
     }
 }
 
+TEST(factory, AuthorityFactory_getAvailableGeoidmodels) {
+    const std::list<std::string> navd88geoidmodels{
+        "GEOID03",  "GEOID06", "GEOID09", "GEOID12A",
+        "GEOID12B", "GEOID18", "GEOID99"};
+
+    const std::string OSGM15{"OSGM15"};
+
+    auto checkNavd88 = [&navd88geoidmodels](const std::list<std::string> &res) {
+        ASSERT_EQ(res.size(), navd88geoidmodels.size());
```

Comment:
this test might be a bit fragile and will break next time we'll add a new US geoid. Can it just test that GEOID12B and GEOID18 are found for example ?